db/setup/queries.sql

```diff
@@ -1,97 +0,0 @@
-INSERT INTO
-  Users ("username", "email", "fullName")
-VALUES
-  (
-    'gbudau-admin',
-    'gbudau-admin@transcendence.com',
-    'George Blue'
-  );
-
-INSERT INTO
-  Users ("username", "email", "fullName")
-VALUES
-  (
-    'vdedios-admin',
-    'vdedios-admin@transcendence.com',
-    'Victor Red '
-  );
-
-INSERT INTO
-  Users ("username", "email", "fullName")
-VALUES
-  (
-    'aollero-admin',
-    'aollero-admin@transcendence.com',
-    'Álvaro Green'
-  );
-
-INSERT INTO
-  Users ("username", "email", "fullName", "id")
-VALUES
-  (
-    'fportela-admin',
-    'fportela-admin@transcendence.com',
-    'Fiz Purple',
-    '75be6ba4-bdc3-45c7-acd7-a5965286750a'
-  );
-
-INSERT INTO
-  Users ("username", "email", "fullName")
-VALUES
-  (
-    'awesome-user',
-    'awesome-user@transcendence.com',
-    'Awesome Gray'
-  );
-
-INSERT INTO
-  Users ("username", "email", "fullName")
-VALUES
-  (
-    'stinky-user',
-    'stinky-user@transcendence.com',
-    'Stinky Brown'
-  );
-
-INSERT INTO
-  Users ("username", "email", "fullName")
-VALUES
-  (
-    'Sporty-user',
-    'sporty-user@transcendence.com',
-    'Sporty Black'
-  );
-
-INSERT INTO
-  Users ("username", "email", "fullName")
-VALUES
-  (
-    'Happy-user',
-    'happy-user@transcendence.com',
-    'Happy Pink'
-  );
-
-INSERT INTO
-  Users ("username", "email", "fullName")
-VALUES
-  (
-    'vde-dios',
-    'vde-dios@transcendence.com',
-    'vde-dios'
-  );
-
-INSERT INTO
-  ChatRoom ("name", "ownerId")
```

Comment:
Non blocking, but we have to seed all other tables we have. We effectively are not seeding chatroom tables anymore, after deleting this file.

Reply:
Thanks! I've created a new task for this #300

Reply:
I'd say we may add a task like this one in every user-story which needs stored data from data-base. I'll add one also for the 1:1 chat US I'm working on